### ISSUE TITLE
"setModelHandling" function doesn't affect vehicles that have been changed using "setElementModel"

### ISSUE THREAD

**Issue by Dutchman101** [multitheftauto/mtasa-blue#1198]
**Describe the bug**
If you change an existing vehicle into another vehicle model using [setElementModel](https://wiki.multitheftauto.com/wiki/SetElementModel), the resulting vehicle will have the default GTA handling rather than any handling set (on the vehicle model you're changing it into) using [setModelHandling](https://wiki.multitheftauto.com/wiki/SetModelHandling).

**To reproduce**

Using the following repro script:
[https://pastebin.com/QNMBxJXS](https://pastebin.com/QNMBxJXS)

1. Spawn initial vehicle with /spawn
2. Replace initial vehicle with /replace
3. Observe unexpected results (as commented in the code snippet)

**Expected behaviour**
If the handling for all vehicle instances of a vehicle model is changed using setModelHandling, then vehicles that became said handling-modified vehicle using setElementModel should also be affected by the changes, rather than loading default GTA handling.

**Version**
Server v1.5.7-release-20360
Client v1.5.7-release-20360

**Additional context**
This bug can affect common scripts such as driving over a vehicle-change "pick up", if the new vehicle is supposed to have modified handling it just won't take effect if done through setElementModel. 
Similarly, vehicle creation menu's or commands that change the vehicle model rather than destroying the old one, spawning a new one and warping the player into it, would be affected.

In order to work around the issue, you'll need to use [setVehicleHandling](https://wiki.multitheftauto.com/wiki/SetVehicleHandling) once the vehicle changed or create a new vehicle rather than re-using the old instance using setElementModel. This should be unnecesary.

**Comment by qaisjp** [comment]
In general I think we need to understand what properties are reset and what are kept. example in a table like this:


| property | reset when model changes? |
| ---------|---------------------------- |
| wheel scale | ????  yes? i think we need to change this. |
| handling | ???? all of these are no, so should it be? |
| colour | no |
| paintjob | no |
| damageproof  | no |
| alpha | no |
| door open state | no |
| headlight colour | no |
| engine state | no |
| nitro level | no |
| sirens | no |
| landing gear down | no |
| blown state | no |
| taxi light on | no |
| paintjob | no |
| plate text | no |

(I'm guessing most of the "no"s above, I haven't tested.)

Basically, if things are not reset for most of the properties, I am not sure it makes sense for the handling properties to be reset.

Even if we decide it should not be reset, we still need to fix vehicle handlings not persisting from the original vehicle.

And it's obviously more complicated when changing between e.g. bike and plane, as not all properties (or handlings) are available for all vehicle kinds.

**Comment by Dutchman101** [comment]
> And it's obviously more complicated when changing between e.g. bike and plane, as not all properties (or handlings) are available for all vehicle kinds

To clarify, the issue is with the vehicle it becomes (turns into, using setElementModel) - if that final vehicle ID has handling applied to it using setModelHandling, there should be no such obstacle as vehicle type to ensure it will load said customized handling and not the GTA default for that final vehicle

Note: if fixing this bug will be done through a refactor of setModelHandling, maybe while at it, also consider this feedback in a related issue: [https://github.com/multitheftauto/mtasa-blue/issues/553#issuecomment-753890665](https://github.com/multitheftauto/mtasa-blue/issues/553#issuecomment-753890665)
